pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -1,8 +1,12 @@
 package v1alpha1
 
 import (
+<<<<<<< 5b2aae465e512f24e1af34fc85b50d55b8d4abda
 	"time"
 
+=======
+	"github.com/gophercloud/gophercloud/openstack/baremetal/v1/nodes"
+>>>>>>> Extend BaremetalHost CRD to support RAID configuration
```

Comment:
Conflicts need to be resolved.

Reply:
Thanks @zaneb. I have resolved the conflicts :)